drivers/platform/x86/asus-wrc.c

```diff
@@ -0,0 +1,134 @@
+/*
+ * Asus Wireless Radio Control Driver
+ *
+ * Copyright (C) 2015 Endless Mobile Inc.
+ *
```

Comment:
Our full legal name is still "Endless Mobile, Inc.".  For now, I recommend we keep that for copyright statements.